assets/src/edit-story/components/checklistCard/index.js

```diff
@@ -16,6 +16,7 @@
 
 export { default as ChecklistCard } from './checklistCard';
 export * from './constants';
-export * as ChecklistCardStyles from './styles';
```

Comment:
Ah, i think this is why the linting is grumpy. Any specific reason to update this to just `*`? Having this named make it easier for the insides of the `checkListCard/Styles` to be generic and grouped.

Reply:
yea I just did it to fix the linter on my local complaining. You mentioned the fix in slack tho and will update to that:
```
import * as ChecklistCardStyles from './styles';

export { ChecklistCardStyles }
```